components/org.wso2.carbon.identity.mgt.store.connector.jdbc/src/main/java/org/wso2/carbon/identity/mgt/store/connector/jdbc/connector/JDBCIdentityStoreConnector.java

```diff
@@ -518,6 +517,50 @@ public IdentityStoreConnectorConfig getIdentityStoreConfig() {
     }
 
     @Override
+    public List<String> getUsers(List<Attribute> attributes, int offset, int length)
+            throws IdentityStoreConnectorException {
+
+        List<String> userIdsToReturn = new ArrayList<>();
+        Map<String, String> properties = identityStoreConfig.getProperties();
+        String databaseType =  properties.get(ConnectorConstants.DATABASE_TYPE);
+        String sqlQuerryForUserAttributes;
+
+        if (databaseType != null && (databaseType.equalsIgnoreCase("MySQL") ||
+                databaseType.equalsIgnoreCase("H2"))) {
+
+            sqlQuerryForUserAttributes = new MySQLFamilySQLQueryFactory()
+                    .getQuerryForUserIdFromMultipleAttributes(attributes, offset, length);
+
+            if (log.isDebugEnabled()) {
+                log.debug("{} sql queries loaded for database type: {}.", sqlQueries.size(), databaseType);
+            }
+        } else {
+            throw new StoreException("Invalid or unsupported database type specified in the configuration.");
+        }
+        try (UnitOfWork unitOfWork = UnitOfWork.beginTransaction(dataSource.getConnection())) {
+
```

Comment:
We have unnecessary empty lines

Reply:
Fixed with https://github.com/wso2-extensions/carbon-security-user-store-jdbc/pull/95